sympy/polys/multivariate_resultants.py

```diff
@@ -0,0 +1,390 @@
+            A sympy polynomial which formulated by Dixon's formulation.
+            Dixon's polynomial is calculated as:
+
+            delta = Delta(A) / ((x_1 - a_1) ... (x_n - a_n)) where,
+
+            A =  |p_1(x_1,... x_n), ..., p_n(x_1,... x_n)|
+                 |p_1(a_1,... x_n), ..., p_n(a_1,... x_n)|
+                 |...             , ...,              ...|
+                 |p_1(a_1,... a_n), ..., p_n(a_1,... a_n)|
+        """
+        if self.m != (self.n + 1):
+            raise ValueError('Method invalid for given combination.')
+
+        # first row
+        rows = [[poly(*self.variables) for poly in self.polynomials]]
+
+        temp = [i for i in self.variables]
+        iterator = iter(self.dummy_variables)
+
+        for idx in (idx for idx in range(self.n)):
+            temp[idx] = next(iterator)
+            rows.append([poly(*temp) for poly in self.polynomials])
+
+        A = sym.Matrix(rows)
+        product_of_differences = functools.reduce(lambda x, y: x * y,
+                                                  [a - b for a, b in zip(self.variables, self.dummy_variables)])
+        dixon_polynomial = (A.det() / product_of_differences).factor()
+        return sym.Poly(dixon_polynomial, *self.dummy_variables)
+
+    def get_coefficients_of_alpha(self, polynomial):
+        r"""
+        Returns
+        --------
+        coefficients: list
+            A list of coefficients (in x_i, ..., x_n terms) of the power products
+            a_1, ..., a_n in Dixon's polynomial
+        """
+        coefficients = []
+        for powers in polynomial.monoms():
+            monomial = functools.reduce(lambda i, j: i * j,
+                                    [a ** b for a, b in zip(self.dummy_variables, powers)])
+            coefficients.append(polynomial.coeff_monomial(monomial))
+
+        return coefficients
+
+    def get_upper_degree(self):
+        list_of_products = [self.variables[i] ** ((i + 1) * self.max_degrees[i] -1)
+                            for i in range(self.n)]
+        product = sym.prod(list_of_products)
+        product = sym.Poly(product).monoms()
+
+        return sym.polys.monomials.monomial_deg(*product)
+
+    def get_dixon_matrix(self, polynomial):
+        r"""
+        Construct the Dixon matrix from the coefficients of polynomial \alpha.
+        Each coefficient is viewed as a polynomial of x_1,..., x_n.
+        """
+        coefficients = self.get_coefficients_of_alpha(polynomial)
+        monomials = list(itermonomials(self.variables, self.get_upper_degree()))
+        monomials = sorted(monomials, key=monomial_key('lex', self.variables))[::-1]
+
+        dixon_matrix = sym.Matrix([[sym.Poly(c, *self.variables).coeff_monomial(m) for m in monomials]
+                                   for c in coefficients])
+
+        keep = [column for column in range(dixon_matrix.shape[-1])
+                if all([elements == 0 for elements in dixon_matrix[:, column]]) is False]
```

Comment:
I think the meaning would be more clear in the form `if any([element != 0 for element in dixon_matrix[:, column]])`